src/messaging/dotnet/src/Abstractions/MessageBufferJsonExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Morgan Stanley makes this available to you under the Apache License,
```

Comment:
The idea was to have the extension methods in the same namespace as the thing they work on. Since the extension method works on the interface in the Abscractions namespace, I moved it there.